cpp/ps0001/p001-soln.cpp

```diff
@@ -1,15 +1,19 @@
 #include <iostream> 
 using namespace std;
 int main() 
-{ int number_one;
+{
+  // prompting user for 3 numbers to add 1/2
+  short number_one;
   cin>>number_one;
-  int number_two;
+  short number_two;
   cin>>number_two;
-  int number_three;
+  short number_three;
   cin>>number_three;
-  int Summation=number_one+number_two+number_three;
+
+  // outputting the result of the summation process 3/3
+  short Summation=number_one+number_two+number_three;
   cout<<"The sumation of three numbers ="<<Summation;

```

Comment:
- [x] did summation process for the three inputted numbers
- [x] used `std::cout` in the right way to display output from the summation process
- [ ] output is as required
```
155
940
168
--------
1263
```
+1 bonus mark as your program is user-friendly ⭐